walkthrough/index.html

```diff
@@ -80,15 +82,16 @@
 
 ## Other
 
-If your output doesn’t match any of the above, click on _I
-don’t know_ below and we could recommend steps to help you.
+If your output does not match any of the above, try the 
+snap package or talk to a volunteer if snap is not an option for you.
 
 ---
 
 ## Do you know your system’s process architecture?
 
 - [[Yes-&gt;Linux tarball]]
-- [[No-&gt;Talk to a Volunteer]]</tw-passagedata><tw-passagedata pid="7" name="Linux directory" tags="" position=""># Moving the Executable to `~/bin`
+- [[No, I would like to try the snap package-&gt;Linux - Snap]]
+- [[No, I would like talk to a volunteer-&gt;Talk to a Volunteer]]</tw-passagedata><tw-passagedata pid="7" name="Linux directory" tags="" position=""># Moving the Executable to `~/bin`
 
 Once you downloaded and extracted the
```

Comment:
`Once you have downloaded` or `Once you download and extract`

Reply:
This is fixed now